src/Pact/Types/Command.hs

```diff
@@ -87,7 +87,7 @@ import Pact.Types.Scheme (PPKScheme(..), defPPKScheme)
 -- In 'Command ByteString', the 'ByteString' payload is hashed and signed; the ByteString
 -- being the JSON serialization of 'Payload Text', where the 'Text' is the pact code; when
 -- executed this is parsed to 'ParsedCode'.
--- Thus, 'Command (Payload ParsedCode)' is the fully executable specialization.
+-- Thus, 'Command (Payload PublicMeta ParsedCode)' is the fully executable specialization.
```

Comment:
This is incorrect, it would be "'Command (Payload m ParsedCode)' (with m representing platform-specific metadata)"

Reply:
Thanks - I'll correct